hedera-mirror-rest/utils.js

```diff
@@ -713,8 +713,14 @@ const randomString = async (length) => {
   return bytes.toString('hex');
 };
 
+const hexPrefix = '0x';
 const addHexPrefix = (hexString) => {
-  return `0x${hexString}`;
+  if (_.isEmpty(hexString)) {
+    return null;
+  }
+
+  hexString = `${hexString}`;
```

Comment:
Shouldn't reassign method parameters in the method body. Here, maybe it's better to rename the method parameter since it might not be string.

```suggestion
  const hexString = `${hexData}`;
```

Reply:
This is now changed so as not to reassign the parameter variable.